helm/hpcc/templates/_helpers.tpl

```diff
@@ -609,7 +610,8 @@ Pass in a dictionary with root
 */}}
 {{- define "hpcc.checkDataMount" -}}
 {{- if .root.Values.storage.dataStorage.forcePermissions | default false }}
-{{ include "hpcc.changeMountPerms" (dict "root" .root "volumeName" "datastorage" "volumePath" "/var/lib/HPCCSystems/hpcc-data") }}
+{{- $volumeName := printf "%s-pv" .root.Values.storage.dataStorage.plane -}}
```

Comment:
good change, somewhat unrelated to this PR though, fix also needed on other systems that forcePermissions is required.
Should prob. separate out into separate JIRA/PR